package.json

```diff
@@ -51,7 +51,6 @@
   "devDependencies": {
     "@cypress/code-coverage": "^3.10.0-dev.1",
     "@cypress/instrument-cra": "1.4.0",
-    "@cypress/react": "^5.10.0",
```

Comment:
I'm assuming there's no corresponding `yarn.lock` file change because this package is being pulled in by `cypress`? Seems odd that there wouldn't be a lockfile change since this was referencing a pretty old version of `@cypress/react`

Reply:
@mike-plummer this is a good point, I think that it should have been removed from the lock file. I'm having an issue where the `yarn` command won't complete for me because of an architecture issue with puppeteer (at least that's what it looks like).

I think it might not be getting to the step where it updates the lock file. Any chance you could try running `yarn` and see if it updates?

Reply:
@mike-plummer good eye. Yep, that did the trick. I updated the lock file 👍🏻